packages/admin/src/components/Apartment/ApartmentCreate.tsx

```diff
@@ -1,54 +1,348 @@
-    </SimpleForm>
-  </Create>
-)
+const ApartmentCreate: FC<ListProps> = props => {
+  const dataProvider = useDataProvider()
+
+  const [metroStations, setMetroStations] = useState<any[]>([])
+  const [accommodations, setAccommodations] = useState<any[]>([])
+  const [sleepingPlaces, setSleepingPlaces] = useState<any[]>([])
+  const [files, setFiles] = useState<any[]>([])
+
+  useEffect(() => {
+    dataProvider.getList('metro-stations', {
+      pagination: { page: 1, perPage: 20 },
+      sort: { field: 'name', order: 'asc' },
+      filter: {}
+    }).then(({ data }) => setMetroStations(data))
+
+    dataProvider.getList('accommodations', {
+      pagination: { page: 1, perPage: 20 },
+      sort: { field: 'name', order: 'asc' },
+      filter: {}
+    }).then(({ data }) => setAccommodations(data))
+
+    dataProvider.getList('sleeping-places', {
+      pagination: { page: 1, perPage: 20 },
+      sort: { field: 'name', order: 'asc' },
+      filter: {}
+    }).then(({ data }) => setSleepingPlaces(data))
+  }, [dataProvider])
+
+  const handleUploadFile = async (file: FilePondFile, apartmentId: number): Promise<void> => {
+    const result = (dataProvider as any).uploadPhoto({
+      file: file.source,
+      apartmentId
+    })
+
+    file.setMetadata('id', result.id)
+    setFiles([...files, file])
+  }
+
+  const handleDeleteFile = async (file: FilePondFile): Promise<void> => {
+    dataProvider.delete('photos', {
+      id: file.getMetadata()?.id
+    })
+  }
+  return (
+    <Create { ...props }>
+      <SimpleForm>
+
+        <Grid container
+          gap={5}
+          justifyContent="center"
+        >
+          <Grid item
+            md={4}
+          >
+            <BooleanInput label="Показывать объявление"
+              source="isActive"
+              validate={[required()]}
+            />
+            <BooleanInput label="Можно с детьми"
+              source="childrenAllowed"
+              validate={[required()]}
+            />
+            <BooleanInput label="Можно с животными"
+              source="petsAllowed"
+              validate={[required()]}
+            />
+
+            <DateTimeInput fullWidth
+              label='Время начала заселения'
+              source='checkInStart'
+              validate={[required()]}
+            />
+            <DateTimeInput fullWidth
+              label='Время конца заселения'
+              source='checkInEnd'
+              validate={[required()]}
+            />
+            <DateTimeInput fullWidth
+              label='Время конца выезда'
+              source='checkOutEnd'
+              validate={[required()]}
+            />
+            <SelectInput fullWidth
+              choices={Object.entries(Term).map(([id, name]) => ({ id, name }))}
+              label='Тип'
+              source='type'
+              validate={[required()]}
+            />
+            <NumberInput fullWidth
+              label='Не больше Н взрослых'
```

Comment:
```suggestion
              label='Не больше N взрослых'
```
Или максимальное число взрослых.